.claude/commands/onboard.md

```
@@ -90,16 +128,81 @@ docker info > /dev/null 2>&1
+   typo makes debugging painful later."
+3. MANUAL: "add yourself as an Authorized User (your Northeastern email) and
+   assign yourself the roles you need. For local dev, assigning all roles is
+   fine."
+
+### 6. Generate the Strapi API token
+
+This cannot be scripted — the contributor has to click through the admin UI.
+Record these steps as MANUAL (with the exact click path so they don't need
+to open the README):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)